packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -466,18 +460,15 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
         if (props.multiselectable) {
             return null;
         }
-        const isItemSelected: boolean = state.selectedItemIndex !== 0;
+        const labelledBy: string = `${this.props.labelledBy} ${triggerId}`;
```

Comment:
Can you include a test that ensures that both of these items are part of the labeled by prop? You could do something like:
```
expect(propStringForLabelledBy.split(" ")).toEqual([labelledById, triggerId]);
```

Reply:
Added a test along those lines to the trigger aria test cases.